src/common/errors.js

```diff
@@ -1,7 +1,18 @@
 'use strict';
 const util = require('util');
 
-class RippleError extends Error {
+// this is needed because extending builtins doesn't work in babel 6.x
+function extendableBuiltin(cls) {
+  function ExtendableBuiltin() {
+    cls.apply(this, arguments);
+  }
+  ExtendableBuiltin.prototype = Object.create(cls.prototype);
+  Object.setPrototypeOf(ExtendableBuiltin, cls);
+
+  return ExtendableBuiltin;
+}
+
+class RippleError extends extendableBuiltin(Error) {
```

Comment:
The variant I posted above doesn't work under nodev4, but compiles successfully using Babel. I think we should just go with your variant for now until we can drop babel altogether :)